src/Microsoft.MobileBlazorBindings.Authentication/Services/AccessTokenProviderAccessor.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.DependencyInjection;
+
+[assembly: InternalsVisibleTo("Microsoft.MobileBlazorBindings.Authentication.Msal")]
```

Comment:
Please put this attribute to AssemblyInfo.cs instead

Reply:
This is one of these SDK projects that generates AssemblyInfo. I'm not really keen on turning off that generation for a single attribute.

Reply:
Could you clarify which generation is that about?

Reply:
https://compiledexperience.com/blog/posts/assemblyinfo-generation

Reply:
Technically I could either turn generation off, and add all assembly attributes, or I could add an AssemblyInfo.cs but NOT add the assembly attributes and keep generation on. However I prefer having the attribute in the class that needs it then. I'll let @Eilon decide.

Reply:
Yep, you don't need to disable generation to add additional attributes to AssemblyInfo.